client/src/app/submission/submit/submit.component.html

```diff
@@ -28,7 +28,7 @@ <h3>What's next?</h3>
   <mat-checkbox
     [(ngModel)]="triggersAnalysisCheck"
     labelPosition="after">
-    Check if you want this submission to be indexed and trigger analysis.
+    Check if you want this submission to be indexed and trigger analysis. Only uncheck this if you really know what you're doing.
```

Comment:
I would add something like "unchecking this will make the resulting dataset unsearchable, and analysis pipeline won't be triggered"

Reply:
I have some changes to the UI. I'll add that.